include/swift/AST/DiagnosticsSIL.def

```diff
@@ -356,6 +356,10 @@ ERROR(embedded_swift_existential_type,none,
       "cannot use a value of protocol type %0 in embedded Swift", (Type))
 ERROR(embedded_swift_existential,none,
       "cannot use a value of protocol type in embedded Swift", ())
+ERROR(embedded_swift_metatype_type,none,
+      "cannot use metatype %0 in embedded Swift", (Type))
```

Comment:
A small suggestion based on the diagnostic messages in test-cases. I think this would benefit from use of "of type" after "metatype" because the type it prints in instance type of a metatype, either that or actually wrap the type in a metatype container before producing a diagnostic:

```
cannot use metatype of type 'Int' in embedded Swift`
```

or

```
cannot use metatype 'Int.Type' in embedded Swift
```

Reply:
fixed